ProjectWithUMLDiagram/src/com/beyond/person/core/exceptions/PersonNotFound.java

```diff
@@ -0,0 +1,7 @@
+package com.beyond.person.core.exceptions;
+
+public class PersonNotFound extends InvalidException{
```

Comment:
@AraChilingaryan this one is fine but again you should have BaseException . 
And why PersonNotFound is fine because your project name is Person , and everything which means "something not found should throw this exception"  . Also change name to PersonNotFoundException

Reply:
done